.ci.yaml

```diff
@@ -14,6 +14,19 @@ enabled_branches:
 platform_properties:
   linux:
     properties:
+      build_host: "false"
```

Comment:
Isn't false the default?

Reply:
It should be, however, there was some concerns to how the engine recipe is pulling properties and assigning default values. So I explicitly set it here.

The goal is to remove the default properties in https://cs.opensource.google/flutter/infra/+/main:config/engine_config.star. Afterwards, we only need to look at the recipe and .ci.yaml